src/ecrc-frontend/src/components/base/termsOfUse/TermsOfUse.js

```diff
@@ -1,18 +1,43 @@
 /* eslint-disable react/jsx-one-expression-per-line */
-import React from "react";
+import { Redirect } from "react-router-dom";
+
+import React, { useState } from "react";
 import PropTypes from "prop-types";
 import { Button } from "../button/Button";
-
 import "./TermsOfUse.css";
 
-export default function TermsOfUse({ onClick }) {
+export default function TermsOfUse({
+  onContinueClick,
+  checkFirstBox,
+  checkSecondBox,
+  termOfUseOnScroll,
+  continueBtnEnabled
+}) {
+  const [toHostHome, setToHome] = useState(false);
```

Comment:
this should be `setToHostHome` to not be confused with going to our app's home route which is `/ecrc`